### PR TITLE
added business continuity known issue

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-10292

### CODE REVIEW

File: release_notes/known_issues_continuity.adoc [hunk 1]
```diff
@@ -32,6 +32,22 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 Backup and restore known issues and limitations are listed here, along with workarounds if they are available.
 
+[#open-cluster-namespace-stuck-terminating-state-disabling-multiclusterhub]
+=== The _open-cluster-management-backup_ namespace is stuck in the _Terminating_ state, disabling the component cluster backup in _MultiClusterHub_
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc]: I think we can cut down this ID and title. We don't have to have the whole phrase.

`_open-cluster-management-backup_ namespace stuck component cluster backup disabled`


`#open-cluster-namespace-stuck-disabled-mch` (or even shorter than this)

File: release_notes/known_issues_continuity.adoc [hunk 2]
```diff
@@ -32,6 +32,22 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
...
+//2.10:ACM-10292
+
+The `open-cluster-management-backup` namespace is stuck in the `Terminating` state in the following instances: 
+
+* If you have a cluster hub where you run a {product-title-short} restore operation.
```
Comment (swopebe) [inline release_notes/known_issues_continuity.adoc]: cluster hub?

**Review by swopebe — CHANGES_REQUESTED** [review]
couple things, otherwise looks good!

**Review by swopebe — APPROVED** [review]
Thanks for the quick edits!

File: release_notes/known_issues_continuity.adoc [hunk 3]
```diff
@@ -32,6 +32,21 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
 
 Backup and restore known issues and limitations are listed here, along with workarounds if they are available.
 
+[#open-cluster-namespace-stuck-terminating]
+=== The _open-cluster-management-backup_ namespace is stuck in the _Terminating_ state
+//2.10:ACM-10292
+
+When the `open-cluster-management-backup` namespace is stuck in the `Terminating` state, it disables the component cluster backup in `MultiClusterHub`. The namespace is stuck in the following instances:
```
Comment (birsanv) [inline release_notes/known_issues_continuity.adoc]: replace 39-42 with :

When the `cluster-backup` component is disabled on the `MultiClusterHub` resource, the `open-cluster-management-backup` namespace is stuck in the `Terminating` state if you have a Velero restore resource created by a {product-title-short} restore operation.

**Review by birsanv — CHANGES_REQUESTED** [review]

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**